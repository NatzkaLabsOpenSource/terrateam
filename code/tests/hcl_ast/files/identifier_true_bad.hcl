foo 1
---
foo true
---
foo false
---
foo 1.2
