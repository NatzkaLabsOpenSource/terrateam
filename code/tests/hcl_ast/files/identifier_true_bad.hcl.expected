Expected equals (=) if an attribute or block labels if a block, found a non-string literal.
